charts/sf-datapath-and-archival-gateway/templates/_helpers.tpl

```diff
@@ -15,43 +15,52 @@ archival-grpc.conf: |-
     listen 0.0.0.0:8081 http2;
 
     location /datasetrawquery.DatasetRawQuery {
-      grpc_pass grpc://{{ .Values.global.archivalReleaseName }}-raw-query-exec-controller:8081;
+      set $upstream "grpc://archival-raw-query-exec-controller:8081";
```

Comment:
lets name upstream / backend to specific vars like
datasetrawcontrollerbackend etc.

Specific to the service name